dsnp/broadcast.ts

```diff
@@ -0,0 +1,41 @@
+import { ParquetModel } from "../types/frequency";
+
+const broadcast: ParquetModel = [
+  {
+    // TODO: Is this needed?
+    name: "announcementType",
```

Comment:
Do we want to have the announcement type still in the file? I think so to comply with the DSNP Spec.